packages/eslint-plugin-pf-codemods/src/ruleCustomization.ts

```diff
@@ -16,6 +16,7 @@ export const warningRules = [
   "emptyState-warn-change-structure",
   "formControls-updated-markup",
   "horizontalSubnav-warn-ariaLabel",
+  "jumpLinksItem-warn-prop-markup-updates",
```

Comment:
I'm not sure if this should really be a warning since the `href` now being required will throw build errors.